bin/save-ebuild-env.sh

```diff
@@ -50,7 +50,7 @@ __save_ebuild_env() {
 	done
 	unset x
 
-	unset -f assert __assert_sigpipe_ok \
+	unset -f __pipestatus __assert_sigpipe_ok \
```

Comment:
`assert` should be kept here because it still exists as a banned function.
```suggestion
	unset -f assert __pipestatus __assert_sigpipe_ok \
```

Reply:
Good catch. That was a leftover from a previous version of the patch.